src/arch/Sound/RageSoundDriver_WaveOut.h

```diff
@@ -15,7 +15,7 @@ class RageSoundDriver_WaveOut: public RageSoundDriver
 	RageSoundDriver_WaveOut();
 	~RageSoundDriver_WaveOut();
 	RString Init();
-
+	bool InitSuccess = false;
```

Comment:
Not the biggest fan of hungarian notation, but let's use it here for consistency for now (m_bInitSuccess)

Reply:
Also put it with the private variable below.